src/app/modules/form/admin-form/admin-form.payments.controller.ts

```diff
@@ -223,7 +225,19 @@ export const _handleUpdatePayments: ControllerHandler<
           level: PermissionLevel.Write,
         }),
       )
-      // Step 3: User has permissions, proceed to allow updating of start page
+      .andThen(checkFormIsEncryptMode)
+      // Step 3: Check that the payment form has a stripe account connected
+      .andThen((form) => {
+        if (
+          !!form.payments_channel.publishable_key &&
+          !!form.payments_channel.target_account_id
```

Comment:
```suggestion
          form.payments_channel.channel === PaymentChannel.Unconnected
```
For extensibility to other channels in the future? I don't think we can assume all channels will have publishable keys and target account ids as properties after all

Reply:
Oh yes thanks for reminding me that we have an `Unconnected` channel! [514dd52](https://github.com/opengovsg/FormSG/pull/6258/commits/514dd521742be02699fb4399708566ea6444d414)